tests/test_regression/TestControllers.py

```diff
@@ -247,6 +247,16 @@ def test_edit_test(self):
         """
         self.create_user_with_role(
             self.user.name, self.user.email, self.user.password, Role.admin)
+
+        for _ in range(3):
+            test = RegressionTest(1, '-autoprogram -out=ttxt -latin1 -2',
+                           InputType.file, OutputType.file, 1, 10)
+            test2 = RegressionTest(1, '-autoprogram -out=ttxt -latin1 -2',
+                           InputType.file, OutputType.file, 2, 10)
+            g.db.add(test)
+            g.db.add(test2)
```

Comment:
You need to add these regression tests to categories. like category.regression_tests.append

Reply:
added